examples/5-eks-cluster-with-windows-support/main.tf

```diff
@@ -120,6 +124,9 @@ module "aws_eks_accelerator" {
     }
   }
 
+  # SELF-MANAGED NODE GROUPS
+  enable_self_managed_nodegroups = true # default false
```

Comment:
This won't be needed anymore.This can cause error

Reply:
Removed it.